Resources/Prototypes/_Moffstation/Entities/Mobs/Customization/Markings/resomi.yml

```diff
@@ -1,33 +1,76 @@
+# Head Markings
 - type: marking
-  id: ResomiTailComplex
-  bodyPart: Tail
-  markingCategory: Tail
-  speciesRestriction:
+  id: ResomiFluffHead
+  bodyPart: Head
+  markingCategory: Head
+  speciesRestriction: 
   - Resomi
```

Comment:
Prototypes in this file use
```yaml
speciesRestriction:
- Resomi
```
while other files use

```yaml
speciesRestriction: [ Resomi ]
```
I'd say pick one and be consistent, at least within Resomi-land